lib/utils/desktop_drop_downloader.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:io';
+
+import 'package:flutter/foundation.dart';
+
+import 'package:http/http.dart';
+import 'package:path_provider/path_provider.dart';
+
+abstract class DesktopDropDownloader {
+  const DesktopDropDownloader._();
+
+  static Future<String?> unsupportedUriCallback(String url) async {
```

Comment:
Is it possible to implement this in the desktop_drop package instead? It could be an optional feature there.